README.md

```diff
@@ -52,6 +52,8 @@ Will call `callback` with an array of detailed ad objects.
 
 Values for `locationId` and `categoryId` can be found by performing a search and looking at the POST request parameters or the URL Kijiji redirects to. For example, after setting the location to Ottawa and selecting the "cars & vehicles" category, Kijiji redirects to http://www.kijiji.ca/b-cars-vehicles/ottawa/c27l1700185. The last part of the URL (c27l1700185) is formatted as c[categoryId]l[locationId]. So in this case, `categoryId` is 27 and `locationId` is 1700185.
 
+By default, each query result is scraped. To suppress this behavior and return only the data retrieved by the initial query, set the `scrape` preference to `false`.
```

Comment:
Please show the field above in the prefs example. The name of the field is also potentially misleading. We are still scraping, just not for any detail beyond what is present in the RSS feed. Something like `scrapeInnerAd` would be more descriptive